### ISSUE TITLE
metadata.annotations: Too long: must have at most 262144 characters

### ISSUE THREAD

**Issue by yantriks-edi-bice** [tensorflow/tfx#2079]
Executing on Kubeflow 1.0 a pipeline based off the taxi template runs into the following error in the Evaluator step

This step is in Error state with this message: Pod "generic-tfx-pipeline-2-7f7xv-969131319" is invalid: metadata.annotations: Too long: must have at most 262144 characters

I see a related issue in argo but it seems there's no workaround as it seems to be a Kubernetes enforced limit and a design issue on the part of the client

https://github.com/argoproj/argo-cd/issues/820

**Comment by yantriks-edi-bice** [comment]
Apparently the following 

eval_config = tfma.EvalConfig(
        model_specs=[tfma.ModelSpec(signature_name='eval')], #label_key=features.LABEL_KEY)],
        slicing_specs=[tfma.SlicingSpec()],
        metrics_specs=tfma.metrics.default_binary_classification_specs()
    )

translates into the following and all those thresholds are listed in the yaml

metrics = [
        tfma.metrics.ExampleCount(name='example_count'),
        #tfma.metrics.WeightedExampleCount(name='weighted_example_count'),
        #tf.keras.metrics.BinaryCrossentropy(name='binary_crossentropy'),
        tf.keras.metrics.BinaryAccuracy(name='accuracy'),
        tf.keras.metrics.AUC(name='auc', num_thresholds=10000),
        tf.keras.metrics.AUC(name='auc_precision_recall', curve='PR', num_thresholds=10000),
        tf.keras.metrics.Precision(name='precision'),
        tf.keras.metrics.Recall(name='recall'),
        #tfma.metrics.MeanLabel(name='mean_label'),
        #tfma.metrics.MeanPrediction(name='mean_prediction'),
        #tfma.metrics.Calibration(name='calibration'),
        tfma.metrics.ConfusionMatrixPlot(name='confusion_matrix_plot'),
        tfma.metrics.CalibrationPlot(name='calibration_plot')
    ]

**Comment by rclough** [comment]
Did you even get around this issue somehow? Or was it fixed?
